### PR TITLE
Fix annotate_sv and some function examples

### PR DESCRIPTION
**Pull request by vladimirsouza**
This used to return an error. Now it's fixed. 
```
> sv_df = get_manta_sv(these_sample_ids = "DOHH-2", projection = "hg38")
Using the bundled Manta SV (.bedpe) calls in GAMBLR.data...
Using the bundled metadata in GAMBLR.data...
> annotated_entrez = annotate_sv(sv_data = sv_df,
+                                with_chr_prefix = TRUE,
+                                collapse_redundant = FALSE,
+                                return_as = "bedpe_entrez", 
+                                genome_build = "hg38")
> annotated_entrez
   chrom1    start1      end1 chrom2    start2      end2 name score strand1 strand2 tumour_sample_id
1:   chr8 127735954 127735954  chr14 105647949 105647949    .    84       +       -           DOHH-2
2:   chr8 127735958 127735959  chr14 105647945 105647946    .    83       -       +           DOHH-2
3:  chr14 105863255 105863255  chr18  63126264  63126264    .   103       +       -           DOHH-2
4:  chr14 105913232 105913232  chr18  63126259  63126259    .    91       -       +           DOHH-2
   gene entrez partner   fusion
1:  MYC   4609     IGH  IGH-MYC
2:  MYC   4609     IGH  IGH-MYC
3: BCL2    596     IGH IGH-BCL2
4: BCL2    596     IGH IGH-BCL2
```

And this continues working:
```
> sv_df = get_manta_sv(projection = "hg38")
Using the bundled Manta SV (.bedpe) calls in GAMBLR.data...
Using the bundled metadata in GAMBLR.data...
> annotated_entrez = annotate_sv(sv_data = sv_df,
+                                with_chr_prefix = F,
+                                collapse_redundant = FALSE,
+                                return_as = "bedpe_entrez", 
+                                genome_build = "hg38")
> annotated_entrez
      chrom1    start1      end1 chrom2    start2      end2 name score strand1 strand2
   1:      1 161688841 161688841      3  16468400  16468400    .   133       +       +
   2:     11  65499812  65499812     14 105644570 105644570    .    88       +       -
   3:     11  65499951  65499951     14 105644568 105644568    .   135       -       +
   4:     13  91324291  91324291     14 105745520 105745520    .    90       -       +
   5:     13  91323445  91323445     16  10886782  10886782    .    73       +       +
  ---                                                                                 
1091:      3 189537648 189537650      8 133891172 133891174    .    84       +       -
1092:      3 189537648 189537650      8 133891172 133891174    .   175       +       -
1093:      6  90175690  90175691     12  11555200  11555201    .   110       -       -
1094:      9  37363960  37363963     19  44794830  44794833    .   111       +       +
1095:      9  37364097  37364098     19  44794748  44794749    .    90       -       -
               tumour_sample_id    gene entrez partner        fusion
   1:                  FL2002T1  FCGR2B   2213   RFTN1  RFTN1-FCGR2B
   2:                 15-38154T  MALAT1 378938     IGH    IGH-MALAT1
   3:                 15-38154T  MALAT1 378938     IGH    IGH-MALAT1
   4:                 15-31924T MIR17HG      0     IGH   IGH-MIR17HG
   5:                  SP124973 MIR17HG      0   CIITA CIITA-MIR17HG
  ---                                                               
1091:                      BL41     MYC   4609    BCL6      BCL6-MYC
1092: BLGSP-71-30-00681-01A-01E     MYC   4609    BCL6      BCL6-MYC
1093:                  SP192993    ETV6   2120   BACH2    BACH2-ETV6
1094:                  SP116674    BCL3      0    PAX5     PAX5-BCL3
1095:                  SP116674    BCL3      0    PAX5     PAX5-BCL3
```

### CODE REVIEW

File: R/annotate_sv.R [hunk 1]
```diff
@@ -140,75 +140,80 @@ annotate_sv = function(sv_data,
...
-    }
-    switch(
-      as.character( sum(is_priority_gene) ),
-      "0" = {
-        # both genes have the lowest priority to be oncogene
-        # if they are partner of each other, report this error
-        if( all( !is.na(dup_pair$partner) ) ){
-          message("Warning: There are two rows with the same values, but inverted ones for the `gene` and `partner` columns (i.e. both genes as oncogene and partner of each other). However, neither of them is listed as a gene with high priority to be oncogene (included in the `priority_to_be_oncogene` object). See these rows below:")
+  # fix duplicated rows (rows with the same marker)
+  if( nrow(all.annotated$duplicate) > 0 ){
```
Comment (vladimirsouza) [inline R/annotate_sv.R:161]: For clarification, I added this `if` here and moved a chunk of code inside it.

File: R/supplement_maf.R [hunk 2]
```diff
@@ -16,8 +16,8 @@
 #' my_metadata = GAMBLR.data::gambl_metadata
 #' reddy_meta = dplyr::filter(my_metadata, cohort=="dlbcl_reddy")
 #'
-#' small_maf = GAMBLR.data::sample_dat$maf$grch37 %>%
-#'  dplyr::filter(Tumor_Sample_Barcode %in% reddy_meta$Tumor_Sample_Barcode)
+#' small_maf = dplyr::filter(GAMBLR.data::sample_data$grch37$maf,
```
Comment (mattssca) [inline R/supplement_maf.R]: Why not update this example to instead call `get_ssm_by_samples` and give the reddy_meta to `these_samples_metadata` parameter? I think this is a better approach than to fetch SSM by loading the `sample_data` in GAMBLR.data.

**Review by mattssca — COMMENTED** [review]
`annotate_sv` looks good and my example now works as intended. One small comment about the `supplement_maf` examples. Let me know if it makes sense. Thanks!

File: R/supplement_maf.R (hunk 2 above)
Comment (mattssca) [reply R/supplement_maf.R]: Vladimir, do you want to address this example? So we can merge this PR as well.

Comment (vladimirsouza) [reply R/supplement_maf.R]: Yes. Thank you. I'll do that.

Comment (vladimirsouza) [reply R/supplement_maf.R]: Done.

**Review by mattssca — APPROVED** [review]
Thanks Vladimir,

**Pull request merged**